src/core/base64.cpp

```diff
@@ -54,4 +54,136 @@ QVariant decode( const QString & _b64, QVariant::Type _force_type )
 }
 
 
+//TODO C++20: It *may* be possible to make the following functions constepxr given C++20's constexpr std::string.
+
+/**
+ * @brief Base64 encodes data.
+ * @param data
+ * @return std::string containing the Base64 encoded data.
+ */
```

Comment:
These doxygen comments may be more useful in the header file rather than the ``.cpp`` file as it is more easily accessible there.

PS: I know I was the one who added the C++20 TODO comment, but maybe integrate it better into the main doxygen comment with ``@todo``.